package.json

```diff
@@ -31,11 +31,11 @@
   "dependencies": {
     "@babel/plugin-transform-runtime": "7.12.1",
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-enterprise-catalog-search": "4.2.0",
-    "@edx/frontend-enterprise-hotjar": "1.3.0",
-    "@edx/frontend-enterprise-logistration": "3.2.0",
-    "@edx/frontend-enterprise-utils": "3.2.0",
-    "@edx/frontend-platform": "4.0.1",
+    "@edx/frontend-enterprise-catalog-search": "5.0.0",
+    "@edx/frontend-enterprise-hotjar": "2.0.0",
+    "@edx/frontend-enterprise-logistration": "4.0.0",
+    "@edx/frontend-enterprise-utils": "4.0.0",
+    "@edx/frontend-platform": "5.0.0",
```

Comment:
Updated dependencies as part of upgrade to react router v6 and resolved a majority of the conflicts.

Update to the `learner-credit-management` routes still required to continue upgrade to v6